addons/observability/metadata.yaml

```diff
@@ -7,7 +7,9 @@ tags:
 
 deployTo:
   control_plane: true
-  runtime_cluster: false
+  runtime_cluster: true
+  customize_runtime_cluster: true
```

Comment:
What's the  meaning  of this filed?

Reply:
Use deployTo.customize_runtime_cluster in add-on metadata
to mark it will need to customize runtime clusters to enable
the addon